biobalm/succession_diagram.py

```diff
@@ -84,6 +89,16 @@ class SuccessionDiagram:
     for smaller networks, the trade-off is in favor of computing a smaller NFVS.
     """
 
+    MAX_MOTIFS_PER_NODE = 100_000
```

Comment:
Maybe this should be a member variable instead of a class variable. For example, if we read in a precomputed SD and it hit this limit, we would want to know what the limit was, right?